src/controllers/activity.controller.js

```diff
@@ -25,7 +25,18 @@ const findAll = (req, res) => {
       if (err) {
         res.json({ success: false, error: err.message });
       } else {
-        res.json({ success: true, data: activities });
+        const data = activities.map(activity => ({
+          id: activity._id, // eslint-disable-line no-underscore-dangle
+          name: activity.name,
+          startDate: new Date(activity.startDate),
```

Comment:
A note for the future, please add a bit more detail in the PR description of why you're doing this map so it's easy to review and understand.